ertviz/ertapi/ensemble/request_data.py

```diff
@@ -0,0 +1,123 @@
+from collections import namedtuple
+import ertviz.ertapi.ensemble
+import pandas as pd
+
+
+class ParametersDict:
+    def __init__(self, field, key, parent):
+        self._field = field
+        self._parent = parent
+        self._key = key
+        self._index = -1
+
+    def __iter__(self):
+        return self
+
+    def _get_gen_class(self):
+        return {
+            "parameters": ertviz.ertapi.ensemble.Parameter,
```

Comment:
Not necessary to create this dict every time

Reply:
True, result of a quick refactoring yesterday :)